ruby/example_code/s3/scenario_getting_started.rb

```diff
@@ -31,7 +31,7 @@ def create_bucket
     bucket = @s3_resource.create_bucket(
       bucket: "doc-example-bucket-#{Random.uuid}",
       create_bucket_configuration: {
-        location_constraint: "us-east-2" # Note: only certain regions permitted
+        location_constraint: "us-east-1" # Note: only certain regions permitted
```

Comment:
There is a Ruby file in this Python PR?

Reply:
It was causing my tests to fail and was uncovered during my testing. Boy scout clause!